migtests/scripts/functions.sh

```diff
@@ -326,3 +347,24 @@ tail_log_file() {
 		echo "No ${log_file_name} found."
 	fi	
 }
+
+kill_process() {
+	to_be_killed=$1
+	kill -9 ${to_be_killed}
```

Comment:
kill -9 might be overkill with the first time itself. Try -15 (and if it doesn't die after a timeout, do -9). For now, maybe just -15 will suffice. 

Problem with -9 is that it doesn't let the process clean up/shutdown child processes, etc.

Reply:
Okk Got it. Thanks!